### PR TITLE
chore: Add Prettier

### PR DESCRIPTION
**Pull request by willdurand**
We shouldn't have to think or argue about codestyle so let's use Prettier like in most other projects we maintain.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -195,6 +195,7 @@ jobs:
           condition:
             equal: [*nodejs_current, << parameters.nodejs >>]
           steps:
+            - run: npm run prettier-ci
```
Comment (willdurand) [inline .circleci/config.yml:198]: This works as shown here 🤣 https://app.circleci.com/pipelines/github/mozilla/web-ext/1653/workflows/f4c65822-c0c2-4d21-8a50-9a7600c3648c/jobs/6694

**Comment by willdurand** [comment]
I just rebased this PR.

**Comment by willdurand** [comment]
rebased!

**Review by rpl — APPROVED** [review]
lgtm thanks Will, I'm going to merge this in a minute.

I think it may be worth adding a mention to the prettier npm script to run before submitting a PR in CONTRIBUTING.md, but let's not do that in this PR, which has already updated the CONTRIBUTING .md file due to prettier reformatting it.

As a side note: I updated the PR title from feat to chore, given that this is an internal rework of the js code and doesn't need to be called out as a feature in the next release changelog.

**Pull request merged**